### PR TITLE
Update to TTD Readme & enable auto-trace config.

### PR DESCRIPTION
**Pull request by mrkmarron**
Update adds support for explicit disable of auto-trace emit from command line. Requires update from ChakraCore release/1.9 (PR #5202).

### CODE REVIEW

File: src/node.cc [hunk 1]
```diff
@@ -3616,6 +3617,8 @@ static void PrintHelp() {
          "  --replay-debug=dir         replay and debug using recording log\n"
          "  --break-first              break at first statement when running\n"
          "                             in --replay-debug mode\n"
+         "  --disableAutoTrace       disable the auto-trace feature during\n"
```
Comment (MSLaguana) [inline src/node.cc]: Is there a tabs/spaces issue here? The explanation text doesn't seem to be aligned with the other options.

**Review by MSLaguana — APPROVED** [review]

File: src/node.cc (hunk 1 above)
Comment (mrkmarron) [reply src/node.cc]: Not sure but fixed and updated PR.

File: src/node.cc [hunk 2]
```diff
@@ -3947,6 +3950,8 @@ static void ParseArgs(int* argc,
     } else if (strstr(arg, "--record-history=") == arg) {
       const char* historyStr = arg + strlen("--record-history=");
       s_ttdSnapHistoryLength = (uint32_t)atoi(historyStr);
+    } else if (strcmp(arg, "--disableAutoTrace") == 0) {
```
Comment (kfarnung) [inline src/node.cc]: Should this be `--disable-auto-trace` to align with other arguments?

**Review by kfarnung — APPROVED** [review]

File: src/node.cc (hunk 2 above)
Comment (mrkmarron) [reply src/node.cc]: Good call. I updated the PR with the "-" version.

**Comment by kfarnung** [comment]
CI: https://ci.nodejs.org/job/chakracore-test-pull-request/264/

File: lib/trace_mgr.js [hunk 3]
```diff
@@ -489,13 +489,13 @@ function generateFuzzyStack(eventKind) {
       return directIsAbsolute(frame);
     });
 
-  const lastframe = errstk.length > 0 ?
-    errstk[errstk.length - 1] :
-    'empty_stack';
-
-  const fname = lastframe.substr(lastframe.lastIndexOf(path.sep) + 1)
+  let fname = "empty_stack";
```
Comment (kfarnung) [inline lib/trace_mgr.js]:
```
16:41:21 not ok 1 - /usr/home/iojs/build/workspace/node-test-linter/lib/trace_mgr.js
16:41:21   ---
16:41:21   message: Strings must use singlequote.
16:41:21   severity: error
16:41:21   data:
16:41:21     line: 492
16:41:21     column: 15
16:41:21     ruleId: quotes
16:41:21   …
```

Comment (mrkmarron) [reply lib/trace_mgr.js]: Fixed.

**Comment by kfarnung** [comment]
Landed as 5c2d217af963677e1bc3d13682efa1a52d1da6f2

**Pull request closed without merging**